### PR TITLE
Update CHANGES and CONTRIBUTORS

### PR DESCRIPTION
**Pull request by hugovk**
For PR https://github.com/urllib3/urllib3/pull/1429.

Also update some HTTP links to HTTPS.

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -259,8 +259,11 @@ In chronological order:
   * Improve contribution guide
   * Add ``HTTPResponse.geturl`` method to provide ``urllib2.urlopen().geturl()`` behavior
 
-* Bruce Merry <http://www.brucemerry.org.za>
+* Bruce Merry <https://www.brucemerry.org.za>
   * Fix leaking exceptions when system calls are interrupted with zero timeout
 
+* [Hugo van Kemenade] <https://github.com/hugovk>
```
Comment (SethMichaelLarson) [inline CONTRIBUTORS.txt]: Remove the square braces around your name

File: CHANGES.rst [hunk 2]
```diff
@@ -11,6 +11,8 @@ dev (master)
 * Add a server_hostname parameter to HTTPSConnection which allows for
```
Comment (SethMichaelLarson) [inline CHANGES.rst]: Could you move the "... <Example>" changelist entry to the bottom of the list? I don't know how it crept up there! :)

File: CHANGES.rst [hunk 3]
```diff
@@ -11,6 +11,8 @@ dev (master)
...
   overriding the SNI hostname sent in the handshake. (Pull #1397)
 
+* Drop support for EOL Python 2.6 (Pull #1429)
```
Comment (SethMichaelLarson) [inline CHANGES.rst]: Change this to (Pull #1429 and Pull #1430) as @pquentin removed a little more Python 2.6 code as well.

Comment (hugovk) [reply CHANGES.rst]: Thanks, I've updated these three things.

**Pull request merged**